assignment_3.py

```diff
@@ -0,0 +1,146 @@
+#creating the class
+class Calc:
+    def __init__(self,num_a,num_b) :  #__init__ is instance method(newly create the objects)
+      self.num_a=num_a
+      self.num_b=num_b
+      
+    def addition(self):
+      return self.num_a+self.num_b
+    
+    def subtraction(self):
+      return self.num_a-self.num_b 
+    
+    def multiply(self):
+      return self.num_a*self.num_b
+    
+    def divition(self):
+      return self.num_a/self.num_b
+    
+value=Calc(2,4)  
+print(value.addition(),"add")  
+print(value.subtraction(),"sub")  
+print(value.multiply(),"mul")  
+print(value.divition(),"div")  
+
+
+
+"""
+6 add
+-2 sub
+8 mul
+0.5 div
+"""
+
+#another example of class
+class MyName:
+  def __init__(self,name,age):
+     self.name=name
+     self.age=age
+     
+details=MyName("rahaman",19)
+print(details.name)
+print(details.age)
+
+
+"""     
+rahaman
+19
+"""
+
+
+
+#creating class in employee details
+class Employee:
```

Comment:
Add more logics to the code. use type, isintance, list, tuple, dict. create like a story. currently you are just printing the data. this won't be enough to show to companies.